encrypted-media/util/utils.js

```diff
@@ -225,4 +225,36 @@ function verifyKeyStatuses(keyStatuses, keys)
     });
 }
 
+// For Clear Key, the License Format is a JSON Web Key (JWK) Set, which contains
+// a set of cryptographic keys represented by JSON. These helper functions help
+// wrap raw keys into a JWK set.
+// See:
+// https://w3c.github.io/encrypted-media/#clear-key-license-format
+// http://tools.ietf.org/html/draft-ietf-jose-json-web-key
+//
+// Creates a JWK from raw key ID and key.
+// |keyId| and |key| are expected to be ArrayBufferViews, not base64-encoded.
+function createJWK(keyId, key)
+{
```

Comment:
Is this needed as a utility function or should be it part of the clearkey-messagehandler.js ?